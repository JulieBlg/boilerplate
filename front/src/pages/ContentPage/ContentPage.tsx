import React from 'react';

import MenuLayout from '../../layouts/MenuLayout';

const ContentPage = () => {
  return (
    <MenuLayout>
      <p>Content Page</p>
    </MenuLayout>
  );
};

export default ContentPage;
