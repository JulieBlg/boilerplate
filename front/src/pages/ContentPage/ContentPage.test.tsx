import React from 'react';
import { render } from '@testing-library/react';

import ContentPage from './ContentPage';

describe('ContentPage', () => {
  it('should render properly', () => {
    const { container } = render(<ContentPage />);

    expect(container).toMatchSnapshot();
  });
});
