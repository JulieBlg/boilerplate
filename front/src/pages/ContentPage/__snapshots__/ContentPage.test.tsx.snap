// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentPage should render properly 1`] = `
<div>
  <p>
    Here is menu layout
  </p>
  <p>
    Content Page
  </p>
</div>
`;
